src/errors.rs

```diff
@@ -35,6 +35,42 @@ pub enum PrimaBridgeError {
     Utf8Error { source: Utf8Error },
 }
 
+impl From<reqwest::Error> for PrimaBridgeError {
+    fn from(error: reqwest::Error) -> Self {
+        Self::HttpError {
+            url: error.url().cloned().unwrap_or_else(default_url),
```

Comment:
I don't like so much the `default_url`. I think this should be implemented using `TryFrom`